hashlips/config.js

```diff
@@ -0,0 +1,120 @@
+const basePath = process.cwd();
+const { MODE } = require(`${basePath}/constants/blend_mode.js`);
+const { NETWORK } = require(`${basePath}/constants/network.js`);
+
+const network = NETWORK.sol;
+
+// General metadata for Ethereum
+const namePrefix = "PM DAO Solana NFT";
+const description = "PM DAO Solana NFT description";
+const baseUri = "ipfs://NewUriToReplace";
+
+const solanaMetadata = {
+  symbol: "PSN",
+  seller_fee_basis_points: 1000, // Define how much % you want from secondary market sales 1000 = 10%
+  external_url: "https://pmdao.notion.site/",
+  creators: [
+    {
+      address: "DWN2bpoTYKEAurA8axd6z8FuYFrZ9fA9j2KRV4VvFxWj",
+      share: 100,
+    },
+  ],
+};
+
+// If you have selected Solana then the collection starts from 0 automatically
+const layerConfigurations = [
+  {
+    growEditionSizeTo: 143360,
+    layersOrder: [
+      { name: "Background" },
+      { name: "Body" },
+      { name: "Box" },
+      { name: "Eye" },
+      { name: "Bulb cap" },
+    ],
+  },
+];
+
+const shuffleLayerConfigurations = false;
+
+const debugLogs = false;
+
+const format = {
+  width: 512,
+  height: 512,
+  smoothing: false,
+};
+
+const gif = {
+  export: false,
+  repeat: 0,
+  quality: 100,
+  delay: 500,
+};
+
+const text = {
+  only: false,
+  color: "#ffffff",
+  size: 20,
+  xGap: 40,
+  yGap: 40,
+  align: "left",
+  baseline: "top",
+  weight: "regular",
+  family: "Courier",
+  spacer: " => ",
+};
+
+const pixelFormat = {
+  ratio: 2 / 128,
+};
+
+const background = {
+  generate: true,
+  brightness: "80%",
+  static: false,
+  default: "#000000",
+};
+
+const extraMetadata = {};
+
+const rarityDelimiter = "#";
+
+const uniqueDnaTorrance = 10000;
```

Comment:
I know this is needed but worth double checking which combination of generatives are overlapping from the output just in case later.